rpm-ostree-bisect

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/python3
```

Comment:
This binary doesn't exist on RHEL/CentOS...you'll need to use `/usr/bin/python`

Reply:
Ooops....I didn't see the comment in about invoking it via `python2 rpm-ostree-bisect`...please ignore me

Reply:
> Ooops..

:smiley_cat: